_tests/browser_tests/page_objects/page_the-bureau.js

```diff
@@ -0,0 +1,10 @@
+var TheBureauPage = function() {
+  this.get = function() {
+    browser.get( '/the-bureau' );
+  };
+
+  this.pageTitle = function() { return browser.getTitle() };
```

Comment:
This line is duplicated at in each page. We should refactor into a shared utility file.